--- dist/dirname/index.js	2023-08-24 23:14:14
+++ dist/dirname/index-edited.js	2023-08-24 22:43:54
@@ -1,6 +1,4 @@
-import { getModuleType } from "../utils/platform.js";
-const _importDynamic = new Function('modulePath', 'return import(modulePath)');
+import _dirname from "./dirname.mjs";
 export default async function dirname() {
-    let dirname = (await _importDynamic(getModuleType() === 'module' ? './dirname.mjs' : './dirname.cjs')).default;
-    return dirname;
+    return _dirname;
 }
